src/data/ACTIONS/BLM.js

```diff
@@ -34,6 +34,13 @@ export default {
 		onGcd: true,
 		castTime: 3.0,
 	},
+	UMBRAL_SOUL: {
+		id: 999999999159, // ID once known
+		name: 'Umbral Soul',
+		icon: 'https://xivapi.com/i/002000/002653.png', // Needs updating once known, still Freeze's icon
+		onGcd: true,
+		castTime: 2.0,
```

Comment:
US is on GCD without a cast time. See https://imgur.com/CDUU3Nk

Reply:
So it is, I definitely read that tooltip wrong. Thanks for the save, fixed.